GeneralJS/top-product.ts

```diff
@@ -89,6 +89,31 @@ const orders = [
 ];
 
 function topProduct(state: string) {
+  // match state to state code
+  const stateCode = states.find(stateObject => stateObject.state == state)?.stateCode;
+
+  // all products for passed state
+  const stateProducts = orders.filter(order => order.stateCode == stateCode);
+
+  // pull each product
```

Comment:
I feel like I should loop through products and determine most repeats at the same time...we'll see if I push up another commit before our meeting 😬